web/viewer.css

```diff
@@ -1347,6 +1347,7 @@ html[dir='rtl'] .attachmentsItem > button {
 
 .dialog > .row {
   display: table-row;
+  word-break: break-all;
```

Comment:
I'll move the declaration.

( The main reason for adding `word-break: break-all` is to prevent a long sequence of characters (such as a file path) from overflowing. This would be useful for the main dialogs as well, so I put it there. But after testing, I notice that the text doesn't overflow in the document properties dialog. This is because the `p` (containing the text) is not inline (but `table-cell`), has word-wrap `break-word` AND an *explicit* (maximum) width. )